lib/srv/db/mysql/proxy.go

```diff
@@ -275,9 +281,9 @@ func (p *Proxy) waitForOK(server *server.Conn, serviceConn net.Conn) error {
 }
 
 const (
-	// serverVersion is advertised to MySQL clients during handshake.
+	// DefaultServerVersion is advertised to MySQL clients during handshake.
 	//
 	// Some clients may refuse to work with older servers (e.g. MySQL
 	// Workbench requires > 5.5).
-	serverVersion = "8.0.0-Teleport"
+	DefaultServerVersion = "8.0.0-Teleport"
```

Comment:
Is there any reason we always return version 8.0.0? If so can the comment be updated to reflect that.

Otherwise can we update this to match the current version of Teleport?

Reply:
This indicates MySQL protocol version, not Teleport version. We have to return 8.0.0 here for backward compatibility and this PR is providing an option for users to override it if they need Teleport to report higher MySQL protocol version for better interoperability with some GUI clients.